README.markdown

```diff
@@ -308,7 +308,7 @@ else {
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
 
-* There should be no blank lines after an opening brace or before a closing brace.
+* There should be one blank line after an opening brace and none before a closing brace.
```

Comment:
I'm not sure whether the blank line before opening brace should be a hard-and-fast rule. For example, a one-line function probably doesn't need the space at the start. I think the opening space should be left to the developer, however I do agree with no space before the closing brace.